<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Shreyansh Priyadarshi</title> <meta name="author" content="Shreyansh Priyadarshi"> <meta name="description" content="The personal website of Shreyansh Priyadarshi. "> <meta name="keywords" content="Computational Oncology, Stem Cells, Bioinformatics, Machine Learning, Digital Health"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%92%A1&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://iamspriyadarshi.github.io/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Shreyansh</span> Priyadarshi </h1> <p class="desc"><a href="mailto:shreyansh.priyadarshi_asp25@ashoka.edu.in">shreyansh.priyadarshi_asp25@ashoka.edu.in</a></p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/prof_pic-480.webp 480w, /assets/img/prof_pic-800.webp 800w, /assets/img/prof_pic-1400.webp 1400w, " sizes="(min-width: 800px) 231.0px, (min-width: 576px) 30vw, 95vw" type="image/webp"></source> <img src="/assets/img/prof_pic.jpg?c78fccddd8be6957d188242bcb71e6fd" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="prof_pic.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div class="clearfix"> <p>I am currently pursuing a Postgraduate Diploma in Computer Science with a minor in Chemistry at <a href="https://www.ashoka.edu.in" rel="external nofollow noopener" target="_blank">Ashoka University</a>. I previously completed my bachelor’s degree at Ashoka University, majoring in Biological Science and minoring in Computer Science. My research endeavors revolve around the fascinating realm of Computational Biology, where I aim to harness the power of Computer Science to address biological challenges. I’m specifically interested in Computational Oncology, which involves leveraging data-driven methodologies to create innovative solutions for the detection, therapy, and control of cancer. Additionally, I have a keen interest in Computational Drug Discovery, focusing on using computational techniques to identify and develop new therapeutic drugs.</p> <p>Currently, I am working under the supervision of <a href="https://www.shubhasis-haldar-cmt.com/" rel="external nofollow noopener" target="_blank">Prof. Shubhasis Haldar</a> from <a href="https://www.bose.res.in/" rel="external nofollow noopener" target="_blank">S.N. Bose National Centre for Basic Sciences</a>, and <a href="https://debayangupta.com/" rel="external nofollow noopener" target="_blank">Prof. Debayan Gupta</a> from <a href="https://www.ashoka.edu.in" rel="external nofollow noopener" target="_blank">Ashoka University</a>. Beyond academics, I find joy in the game of Pool, a passion that adds balance to my life.</p> </div> <h2><a href="/publications/" style="color: inherit;">selected publications</a></h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div> <div id="shreyansh2025oncomark" class="col-sm-8"> <div class="title">Robust Prediction of Patient-Specific Cancer Hallmarks Using Neural Multi-Task Learning: a model development and validation study</div> <div class="author"> Shreyansh Priyadarshi, Camellia Mazumder, Bhavesh Neekhra, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Sayan Biswas, Debojyoti Chowdhury, Debayan Gupta, Shubhasis Haldar' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>bioarXiv (preprint)</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.biorxiv.org/content/10.1101/2025.02.03.636380v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Cancer progression is driven by a set of well-defined biological principles—collectively termed the “hallmarks of cancer”—yet current diagnostic approaches seldom incorporate these distinct molecular features into clinical practice. Despite substantial progress in molecular oncology, traditional methods like histopathological grading and immunohistochemical assays often fail to capture the complex interplay between cancer cells and the tumor microenvironment, emphasizing the need for robust computational frameworks capable of systematically quantifying hallmark-specific activity. Here, we address this gap by developing OncoMark, a high-throughput neural multi-task learning (N-MTL) framework designed to simultaneously quantify hallmark activities in tumor biopsies using transcriptomics data. We show that OncoMark achieves near-perfect accuracy, precision, recall, and F1 scores (&gt;99%) in cross-validation, with external validation consistently exceeding 96.6% on five independent datasets. Further evaluation on eight additional datasets—including large-scale cancer cohorts (TCGA, MET500, CCLE, TARGET, PCAWG, POG570) and normal tissue datasets (GTEx, ANTE)—demonstrated high specificity for normal samples and robust sensitivity for hallmark prediction in cancer. By delivering a comprehensive and cost-effective molecular portrait of tumor biology and providing a user-friendly web platform accessible at [https://oncomark-ai.hf.space/](https://oncomark-ai.hf.space/), OncoMark has the potential to guide tailored treatment strategies and advance precision oncology. More broadly, this framework signifies a transformative step toward routine hallmark-based diagnostics, promising to improve patient outcomes by facilitating timely and precise tumor profiling.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shreyansh2025oncomark</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Robust Prediction of Patient-Specific Cancer Hallmarks Using Neural Multi-Task Learning: a model development and validation study}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Priyadarshi, Shreyansh and Mazumder, Camellia and Neekhra, Bhavesh and Biswas, Sayan and Chowdhury, Debojyoti and Gupta, Debayan and Haldar, Shubhasis}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioarXiv (preprint)}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1101/2025.02.03.636380}</span><span class="p">,</span>
  <span class="na">metatype</span> <span class="p">=</span> <span class="s">{Under Review}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div> <div id="shreyansh2024stem" class="col-sm-8"> <div class="title">Comprehensive Enumeration of Cancer Stem-like Cell Heterogeneity Using Deep Neural Network</div> <div class="author"> Debojyoti Chowdhury*, Shreyansh Priyadarshi*, Sayan Biswas, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Bhavesh Neekhra, Debayan Gupta, Shubhasis Haldar' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>bioarXiv (preprint)</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.biorxiv.org/content/10.1101/2024.11.26.625418v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Cancer stem cells (CSCs), a distinct subpopulation within tumors, are pivotal in driving treatment resistance and tumor recurrence, posing substantial challenges to conventional therapeutic strategies. Precise quantification and profiling of these cells are essential for improving cancer treatment outcomes. We present ACSCeND, an advanced deep neural network model accompanied by a robust workflow, specifically developed to quantify cellular compositions from bulk RNA-seq data, enabling accurate CSC profiling. By integrating bulk RNA-seq data with insights derived from single-cell RNA-seq datasets, ACSCeND effectively captures the diversity and hierarchical organization of tumor-resident cell states, alongside cell-specific gene expression profiles (GEPs). Compared to current tissue deconvolution models, ACSCeND exhibits superior performance, achieving significantly higher Concordance Correlation Coefficient (CCC) values and lower Root Mean Square Error (RMSE) across various pseudobulk and real-world bulk tissue samples. Application of ACSCeND to TCGA and PRECOG datasets reveals a strong association between CSC abundance and poorer disease-free survival outcomes, underscoring the clinical relevance of CSCs in cancer progression. Furthermore, cell-specific GEPs for distinct CSC states unveil novel molecular signatures and illuminate the origins of CSC-driven tumor heterogeneity. In summary, ACSCeND provides a powerful, scalable platform for high-throughput quantification of cellular compositions and distinct potency states within normal tissues as well as highly heterogeneous tissues, such as tumors.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shreyansh2024stem</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Comprehensive Enumeration of Cancer Stem-like Cell Heterogeneity Using Deep Neural Network}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chowdhury*, Debojyoti and Priyadarshi*, Shreyansh and Biswas, Sayan and Neekhra, Bhavesh and Gupta, Debayan and Haldar, Shubhasis}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{bioarXiv (preprint)}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1101/2024.11.26.625418}</span><span class="p">,</span>
  <span class="na">metatype</span> <span class="p">=</span> <span class="s">{Under Review}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Nature</abbr></div> <div id="shreyansh2024kindlin" class="col-sm-8"> <div class="title">Pan-cancer analyses suggest kindlin-associated global mechanochemical alterations</div> <div class="author"> Debojyoti Chowdhury*, Ayush Mistry*, Debashruti Maity, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Riti Bhatia, Shreyansh Priyadarshi, Simran Wadan, Soham Chakraborty, Shubhasis Haldar' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Nature Communications Biology</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s42003-024-06044-5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Kindlins serve as mechanosensitive adapters, transducing extracellular mechanical cues to intracellular biochemical signals and thus, their perturbations potentially lead to cancer progressions. Despite the kindlin involvement in tumor development, understanding their genetic and mechanochemical characteristics across different cancers remains elusive. Here, we thoroughly examined genetic alterations in kindlins across more than 10,000 patients with 33 cancer types. Our findings reveal cancer-specific alterations, particularly prevalent in advanced tumor stage and during metastatic onset. We observed a significant co-alteration between kindlins and mechanochemical proteome in various tumors through the activation of cancer-related pathways and adverse survival outcomes. Leveraging normal mode analysis, we predicted structural consequences of cancer-specific kindlin mutations, highlighting potential impacts on stability and downstream signaling pathways. Our study unraveled alterations in epithelial–mesenchymal transition markers associated with kindlin activity. This comprehensive analysis provides a resource for guiding future mechanistic investigations and therapeutic strategies targeting the roles of kindlins in cancer treatment.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">shreyansh2024kindlin</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Pan-cancer analyses suggest kindlin-associated global mechanochemical alterations}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Chowdhury*, Debojyoti and Mistry*, Ayush and Maity, Debashruti and Bhatia, Riti and Priyadarshi, Shreyansh and Wadan, Simran and Chakraborty, Soham and Haldar, Shubhasis}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Communications Biology}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1038/s42003-024-06044-5}</span><span class="p">,</span>
  <span class="na">metatype</span> <span class="p">=</span> <span class="s">{Published}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%73%68%72%65%79%61%6E%73%68.%70%72%69%79%61%64%61%72%73%68%69%30%32@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fa-solid fa-envelope"></i></a> <a href="https://orcid.org/0000-0002-6230-4574" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=cftXdQsAAAAJ" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://www.researchgate.net/profile/Shreyansh-Priyadarshi/" title="ResearchGate" rel="external nofollow noopener" target="_blank"><i class="ai ai-researchgate"></i></a> <a href="https://github.com/iamspriyadarshi" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> <a href="https://www.linkedin.com/in/shreyansh-priyadarshi" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-linkedin"></i></a> <a href="https://twitter.com/iamspriyadarshi" title="X" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-x-twitter"></i></a> </div> <div class="contact-note"> The best way to reach me is via email. </div> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2025 Shreyansh Priyadarshi. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: February 08, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?07b8786bab9b4abe90d10e61f7d12ff7" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>